pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/template/base/PensjonLatex.kt

```diff
@@ -415,11 +429,73 @@ object PensjonLatex : BaseTemplate() {
 
             is Element.NewLine ->
                 printWriter.printCmd("newline")
+            is Element.Table ->
+                with(printWriter) {
+                    val cellWidths = element.rows.map { it.cells.sumOf { cell -> cell.cellColumns } }.distinct()
+                    verifyCells(cellWidths)
+
+                    val numberOfColumns = cellWidths[0]
+
+                    print("\\FloatBarrier", escape = false)
+                    printCmd("begin") {
+                        arg { print("tblr") }
+                        arg {
+                            print(
+                                "colspec={${columnFormat(numberOfColumns)}},"
+                                        + "width=\\textwidth", escape = false
+                            )
+                        }
+                    }
+                    printCmd("hline")
+                    element.rows.forEach { row ->
+                        when (row.colour) {
+                            GRAY -> print("\\SetRow{gray9}", escape = false)
+                            WHITE -> {}
+                        }
+
+                        row.cells.forEachIndexed { index, cell ->
+                            print("\\SetCell[c=${cell.cellColumns}]{l}", escape = false)
```

Comment:
Hva er grunnen til at du gjør det på denne måten istedenfor `\multicolumn`?

Reply:
multicolumn er ikke laget for tblr. SetCell tar også inn alignment og er mer justerbar.